src/shared/bus-unit-util.c

```diff
@@ -381,6 +435,56 @@ int bus_append_unit_property_assignment(sd_bus_message *m, const char *assignmen
                         r = sd_bus_message_append(m, "v", "a(st)", 1, path, u);
                 }
 
+        } else if (streq(field, "CPUSchedulingPolicy")) {
+                int n;
+
+                n = sched_policy_from_string(eq);
+                if (n < 0)
+                        return log_error_errno(r, "Failed to parse CPUSchedulingPolicy: %s", eq);
+
+                r = sd_bus_message_append(m, "v", "i", (int32_t) n);
```

Comment:
hmm, so if we expose this in binary form we should probably fix the type of this. Internally in the kernel the securebits are stored as an "unsigned int", and we should hence do that too, and the dbus property should be "u" here... (unsigned maks a lot more sense for flags). Previously the precise type didn't matter too much, since it wasn't visible in the interface, but with this bus API it would become visible, hence we should fix our internal type for this too.